persistence/relational-jdbc/src/test/resources/logback-test.xml

```diff
@@ -19,21 +19,14 @@
  under the License.
 
 -->
-<configuration>
+<configuration debug="false">
```

Comment:
Should the file be removed, since logback isn't there?

Reply:
This module is not a Quarkus module, so it uses Logback. It's OK to have this file.

Reply:
ah, right, ignore this comment